src/Symfony/Component/Stopwatch/StopwatchPeriod.php

```diff
@@ -25,20 +25,21 @@ class StopwatchPeriod
     /**
      * Constructor.
      *
-     * @param int $start The relative time of the start of the period (in milliseconds)
-     * @param int $end   The relative time of the end of the period (in milliseconds)
+     * @param int|float $start         The relative time of the start of the period (in milliseconds)
+     * @param int|float $end           The relative time of the end of the period (in milliseconds)
+     * @param bool      $morePrecision If true, time is stored as float to keep the original microsecond precision
      */
-    public function __construct($start, $end)
+    public function __construct($start, $end, /* bool */ $morePrecision = false)
```

Comment:
I'm skeptical about this practice of adding commented type hints, especially when it just duplicates the docblock